### ISSUE TITLE
When I try to authenticate with MyUSA, I get a 404

### ISSUE THREAD

**Issue by ultrasaurus** [18F/api.data.gov#255]
From https://api.data.gov/admin/login, when I click "Login with MyUSA"

![image](https://cloud.githubusercontent.com/assets/41906/8435754/f2cb539a-1f07-11e5-9e87-81935f08dec7.png)

**Comment by cmc333333** [comment]
Hi @ultrasaurus ! Was this resolved? I know MyUSA was having some issues last week

**Comment by cmc333333** [comment]
This is not resolved -- our MyUSA configuration needs to change.

**Comment by esgoodman** [comment]
Hey @cmc333333 -- have you joined the public MyUSA integrator mailing list? We are here to help :-)

**Comment by cmc333333** [comment]
Temporarily removed MyUSA login. Will add it back once the appropriate app is approved.

@GUI I just manually modified the configs; I'm betting I need to peak into an encrypted secrets file.

**Comment by cmc333333** [comment]
MyUSA now available again due to https://github.com/NREL/api-umbrella-web/issues/21 . Leaving this ticket open, however, as the configs need to be updated in the long-term config storage.

**Comment by GUI** [comment]
Closing, since this was superseded by https://github.com/18F/api.data.gov/issues/268
